psptool/psptool.py

```diff
@@ -122,3 +123,42 @@ def ls_entries(self, entries=None, verbose=False):
             fields += verbose_fields
 
         print(t.get_string(fields=fields))
+
+    def ls_entries_json(self, entries=None, no_duplicates=False, verbose = False):
```

Comment:
this should just be `ls_json` and do the same traversal as `ls`.
The extra params `no_duplicates`  and `verbose` are probably and unnecessary, just print out the full thing. It's meant fkr further processing anyway, so what fields to use can be decided later on :)

Reply:
Yep, will push a commit with the suggested renaming.